description.md

```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
```

Comment:
Nice to think about video games.

Reply:
Recordings are still videos, right? I would word it as "Images, video and audio files, educational video games ..."